apisix/patch.lua

```diff
@@ -86,6 +92,53 @@ do
 end
 
 
+-- Inspired by kong.globalpatches
+do -- `_G.math.randomseed` patch
```

Comment:
why we need this patch?

Reply:
@moonming Please see https://github.com/apache/apisix/issues/5417 for details.